src/Elements/ElementCustomerService.php

```diff
@@ -43,26 +33,44 @@ class ElementCustomerService extends BaseElement
      * @var array
      */
     private static $db = [
-        'Title' => 'Varchar(255)',
```

Comment:
Does this change require a task to move existing data?

Reply:
data seems to map to both tables